docs/src/file_details/binned.rst

```diff
@@ -0,0 +1,95 @@
+Binned catalogs
+===============
+
+Binned catalogs are split with each tomographic bin having its own HDF5 group.
+
+They can also include calibrations, applied such that they can be used directly in 2pt calculations or other science measurements.
+
+
+Binned shear catalogs
+---------------------
+
+The tomographically binned shear catalogs also include the shear calibration to correct for measurement and selection biases. 
+
+The bin index b runs from 0 to nbin-1, or "all" tag for a combined non-tomographic sample. The "all" bin is not simply the same as all the other bins combined, because the selection bias calibration is different.
+
+Like the original source shear catalogs, the binned shear catalogs may include only a subset of magnitude values depending on what was measured during the pipeline.
+
+=======  =======  =========  ==========  =========
```

Comment:
So this is an example of what a file could look like?

Reply:
Yep - I've added a note to this effect.